packages/server/graphql/types/RemoveMattermostAuthPayload.ts

```diff
@@ -0,0 +1,28 @@
+import {GraphQLID, GraphQLNonNull, GraphQLObjectType} from 'graphql'
+import User from './User'
+import {GQLContext} from '../graphql'
+import makeMutationPayload from './makeMutationPayload'
+
+export const RemoveMattermostAuthSuccess = new GraphQLObjectType<any, GQLContext>({
+  name: 'RemoveMattermostAuthSuccess',
+  fields: () => ({
+    teamId: {
+      type: new GraphQLNonNull(GraphQLID),
+      description: 'The team with updated mattermost auth'
+    },
+    user: {
```

Comment:
nit-- i like to offer foreign keys as well as the objects they get on mutation payloads, but that's probably prematurely optimizing for when we make our API public.

e.g. add `team, userId` to the fields

Reply:
I'm going to leave this alone for now, because this mutation will go away on the generalized interface I've cooked up